packages/expo-camera/src/next/CameraView.tsx

```diff
@@ -192,8 +192,8 @@ export default class Camera extends React.Component<CameraProps> {
    * ```
    */
   static useMicrophonePermissions = createPermissionHook({
-    getMethod: Camera.getMicrophonePermissionsAsync,
-    requestMethod: Camera.requestMicrophonePermissionsAsync,
+    getMethod: CameraView.getMicrophonePermissionsAsync,
```

Comment:
I'm wondering if we should move some of these functions out of the `CameraView`.
These permission ones are not necessarily related with the view, so I think we could also expose a `Camera` that would be the module object that exposes some set of methods. Then you don't need to wrap them into static functions of the component 😉

Reply:
Cool, that makes sense 👍